assets/css/src/generic/_forms-checkbox.scss

```diff
@@ -0,0 +1,38 @@
+.mailpoet-form-checkbox {
+  align-items: center;
+  cursor: pointer;
+  display: inline-flex;
+  position: relative;
+  vertical-align: middle;
+
+  input {
+    height: 1px;
+    position: absolute;
+    visibility: hidden;
+    width: 1px;
```

Comment:
I'm not sure which of these reasons, but either for screenreaders (they would skip elements with zero sizes) or some older regular browser version, which would render the checkbox non-interactive if it didn't have any size. There is a chance none of the reasons is still valid, but it doesn't really change anything since there is `visibility: hidden`;

Reply:
Right, I thought maybe we should add a comment so it always stays like that (if it's needed).